### ISSUE TITLE
[Bug]: Package recode

### ISSUE THREAD

**Issue by tathastu871** [termux/termux-packages#8465]
### Problem
   SignFORTIFY: %n not allowed on Android
FORTIFY: %n not allowed on Android

### What steps will reproduce the bug?

curl site | recode html..ascii
curl -ss 'https://www.google.com' | recode html..ascii

### System information

termux-info:

```
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.org/apt/termux-main/ stable main
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.org/apt/termux-x11 x11 main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
Updatable packages:
command-not-found/stable 1.71 aarch64 [upgradable from: 1.70]
curl/stable 7.81.0 aarch64 [upgradable from: 7.80.0]
gettext/stable 0.21-4 aarch64 [upgradable from: 0.21-3]
glib-bin/stable 2.70.2-3 aarch64 [upgradable from: 2.70.2-2]
glib/stable 2.70.2-3 aarch64 [upgradable from: 2.70.2-2]
gnupg/stable 2.3.4 aarch64 [upgradable from: 2.3.3-2]
gpgv/stable 2.3.4 aarch64 [upgradable from: 2.3.3-2]
libcurl/stable 7.81.0 aarch64 [upgradable from: 7.80.0]
libpcreposix/stable 8.45-1 aarch64 [upgradable from: 8.45]
libprotobuf/stable 2:3.19.2 aarch64 [upgradable from: 2:3.19.1]
libunistring/stable 0.9.10-5 aarch64 [upgradable from: 0.9.10-4]
ndk-sysroot/stable 23b-4 aarch64 [upgradable from: 23b-3]
pcre-static/stable 8.45-1 aarch64 [upgradable from: 8.45]
pcre2-static/stable 10.39-2 aarch64 [upgradable from: 10.39-1]
pcre2/stable 10.39-2 aarch64 [upgradable from: 10.39-1]
pcre/stable 8.45-1 aarch64 [upgradable from: 8.45]
proot/stable 5.1.107-51 aarch64 [upgradable from: 5.1.107-50]
sdl2-static/x11 2.0.18 aarch64 [upgradable from: 2.0.10-25]
sdl2/x11 2.0.18 aarch64 [upgradable from: 2.0.10-25]
termux-tools/stable 0.155 all [upgradable from: 0.154]
uchardet/stable 0.0.7 aarch64 [upgradable from: 0.0.7]
Android version:
10
Kernel build information:
Linux localhost 4.4.177-22371317 #1 SMP PREEMPT Tue Sep 28 16:35:24 KST 2021 aarch64 Android
Device manufacturer:
samsung
Device model:
SM-A750F

```

**Comment by Grimler91** [comment]
Please provide an example that reproduces the issue, I don't get the error with for example termux.com:
```
$ curl termux.com | recode html..ascii
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
```

**Comment by tathastu871** [comment]
![Screenshot_20220106-124648_Termux](https://user-images.githubusercontent.com/65007212/148344099-d9db05c2-5a2d-45d2-a0fc-35cbb9a27d3f.jpg)
![Screenshot_20220106-124651_Termux](https://user-images.githubusercontent.com/65007212/148344114-43e627b2-0a64-49b4-a82f-d187a43f5c87.jpg)

**Comment by Grimler91** [comment]
Hopefully fixed in 3.7.9-2: https://github.com/termux/termux-packages/commit/3d88c20ea2157d38a0d37fec33b13eac529024c3
